app/scripts/migrations/134.test.ts

```diff
@@ -0,0 +1,56 @@
+import { migrate, version } from './134';
+
+const oldVersion = 133;
+
+describe(`migration #${version}`, () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('Does nothing if `usedNetworks` is not in the AppStateController state', async () => {
+    const oldState = {
+      AppStateController: {
+        timeoutMinutes: 0,
+      },
+    };
+
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: oldState,
```

Comment:
Nit: Both here and in the next test, if we deep clone the state before passing it in, it will be more effective in catching false positive test results caused by state mutation.

Reply:
Oh actually, this is not a nit after all, this is causing the 3rd test to be a false positive (the assertion is wrong)

Reply:
done

Reply:
It looks like you've used a shallow clone, which would suffer from the same problem. The assertion doesn't work because the object you're comparing is the same as the one in the migration that has the property deleted from it.

Try using `cloneDeep` from lodash instead